tests/Predis/Command/Redis/BloomFilter/BFADD_Test.php

```diff
@@ -61,6 +61,7 @@ public function testParseResponse(): void
 
     /**
      * @group connected
+     * @group relay-incompatible
```

Comment:
Why all these `@group relay-incompatible`? They should be able to be called using `rawCommand()`.

Reply:
@vladvildanov: How about we call all the newly added groups `relay-resp3` and ignore it too. That will make it easier to "undo" this change without having to sort through the groups.

Reply:
@tillkruss Agreed

Reply:
@tillkruss Fixed